lib/periodic_jobs.rb

```diff
@@ -43,6 +43,8 @@
   # Updates status of FormSubmissions per call to Lighthouse Benefits Intake API
   mgr.register('0 2 * * 0', 'Form526StatusPollingJob', 'args' => { paranoid: true })
   # Checks all 'success' type submissions in LH to ensure they haven't changed
+  mgr.register('5 4 * * 7', 'Form526StateLoggingJob')
+  # Log the state of Form 526 submissions to hydrate Datadog monitor
```

Comment:
The comment descrbing the job should be before the job.

Reply:
I believe Sam was just following convention in this file, which has, confusingly, included a description **after** the job.

Reply:
You're right. That's messed up. Haha. I'll make a PR.